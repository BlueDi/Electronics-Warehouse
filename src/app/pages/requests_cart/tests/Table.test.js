import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import RequestsTable from '../Table';

var item = {
  description: 'BMW motor',
  details: { details: 'Super detailed description', manufacturer: 'Multicomp' },
  free_stock: 1,
  image:
    'iVBORw0KGgoAAAANSUhEUgAAAG8AAABYCAYAAAD7uql1AAAgAElEQVR4Xu29e5RV5Znu+5vfN6/r\nVmtVFXWjoChABFERVCKgUUAjthFzMZqYxLjT3aP79D456d49zj5nZJzTo0f/0+fsHumT7O6xY/be\n2umklcSEaFRuIiiIF0AUFQVFQSAlRRVVRdW6zut3/phrLsâ€¦',
  last_edit: '2018-11-20T11:46:01.628Z',
  last_price: 89.12,
  location: 'HHD12U',
  reference: '213748227'
};

var columns = [
  { name: 'description', title: 'Description' },
  { name: 'image', title: 'Image' },
  { name: 'total_stock', title: 'Total Stock' },
  { name: 'free_stock', title: 'Free Stock' },
  { name: 'last_price', title: 'Last Price' },
  { name: 'location', title: 'Location' },
  { name: 'user_comments', title: 'User Comments' },
  { name: 'details', title: 'Details' },
  { name: 'reference', title: 'Reference' },
  { name: 'packaging_id', title: 'Packaging ID' },
  { name: 'last_edit', title: 'Last Edit' },
  { name: 'properties', title: 'Properties' },
  { name: 'category', title: 'Category' }
];

describe('RequestsTable', () => {
  it('should be defined', () => {
    expect(RequestsTable).toBeDefined();
  });

  it('renders properly', () => {
    const component = renderer.create(
      <Router>
        <RequestsTable items={[item]} columns={columns} />
      </Router>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
