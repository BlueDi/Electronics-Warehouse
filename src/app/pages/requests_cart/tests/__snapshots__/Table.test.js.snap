// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequestsTable renders properly 1`] = `
<div
  className="RootBase-root-1"
>
  <div
    className="TableContainer-root-2"
  >
    <table
      className="MuiTable-root-7 Table-table-3"
      style={
        Object {
          "minWidth": "1608px",
        }
      }
    >
      <colgroup>
        <col
          style={
            Object {
              "width": "48px",
            }
          }
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
        <col
          style={null}
        />
      </colgroup>
      <thead
        className="MuiTableHead-root-8"
      >
        <tr
          className="MuiTableRow-root-9 MuiTableRow-head-12"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableStubCell-cell-14"
            colSpan={1}
            scope="col"
          />
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Description
                  </span>
                  <i
                    aria-hidden="true"
                    className="arrow up icon"
                  />
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Image
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Total Stock
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Free Stock
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Last Price
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Location
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    User Comments
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Details
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Reference
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Packaging ID
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Last Edit
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Properties
                  </span>
                </button>
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableHeaderCell-cell-29 TableHeaderCell-cellNoWrap-39"
            colSpan={1}
            scope="col"
            style={null}
          >
            <div
              className="TableHeaderCell-container-38"
            >
              <div
                className="Content-content-40"
              >
                <button
                  className="ui fluid icon right labeled button"
                  onClick={[Function]}
                >
                  <span
                    className="Title-title-43"
                  >
                    Category
                  </span>
                </button>
              </div>
            </div>
          </th>
        </tr>
        <tr
          className="MuiTableRow-root-9 MuiTableRow-head-12"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableStubCell-cell-14"
            colSpan={1}
            scope="col"
          />
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-head-17 TableFilterCell-cell-44"
            colSpan={1}
            scope="col"
          >
            <div
              className="TableFilterCell-flexContainer-45"
            >
              <div
                className="MuiInputBase-root-60 MuiInput-root-47 MuiInput-underline-51 MuiInputBase-fullWidth-69 MuiInput-fullWidth-54"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input-70 MuiInput-input-55 Editor-input-46"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Filter..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </th>
        </tr>
      </thead>
      <tbody
        className="MuiTableBody-root-77"
      >
        <tr
          className="MuiTableRow-root-9"
          onClick={[Function]}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableDetailToggleCell-toggleCell-78"
            colSpan={1}
            style={null}
          >
            <button
              className="MuiButtonBase-root-86 MuiIconButton-root-80 TableDetailToggleCell-toggleCellButton-79"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="MuiIconButton-label-85"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-89"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root-103"
              />
            </button>
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            BMW motor
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            iVBORw0KGgoAAAANSUhEUgAAAG8AAABYCAYAAAD7uql1AAAgAElEQVR4Xu29e5RV5Znu+5vfN6/r
VmtVFXWjoChABFERVCKgUUAjthFzMZqYxLjT3aP79D456d49zj5nZJzTo0f/0+fsHumT7O6xY/be
2umklcSEaFRuIiiIF0AUFQVFQSAlRRVVRdW6zut3/phrLs…
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          />
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            1
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            89.12
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            HHD12U
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          />
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            <p>
              details: Super detailed description
            </p>
            <p>
              manufacturer: Multicomp
            </p>
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            213748227
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          />
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          >
            2018-11-20T11:46:01.628Z
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          />
          <td
            aria-sort={null}
            className="MuiTableCell-root-16 MuiTableCell-body-18 TableCell-cell-98 TableCell-cellNoWrap-102"
            colSpan={1}
          />
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
